internal/contour/cluster.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2018 Heptio
+// Copyright © 2019 Heptio
```

Comment:
2019?

Reply:
There's some key combination that I hit in vim that autoincrements the first thing that looks like a number. I'll fix this and merge.